LibTest/async/MultiStreamController/isPaused_A01_t02.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion bool get isPaused
+/// Whether the subscription would need to buffer events.
+///
+/// This is the case if the controller's stream has a listener and it is paused,
+/// or if it has not received a listener yet. In that case, the controller is
+/// considered paused as well.
+///
+/// A broadcast stream controller is never considered paused. It always forwards
+/// its events to all uncanceled subscriptions, if any, and let the
+/// subscriptions handle their own pausing and buffering.
```

Comment:
Might want to say that a `MultiStreamController` is not a broadcast stream controller, even if its stream has `isBroadcast` as `true`. 
(It's just a controller for a single stream subscription on a stream which claims to be a broadcast stream, so it should cooperate with other controllers for other subscriptions on the same stream about acting like a broadcast stream.)

Reply:
https://github.com/dart-lang/sdk/issues/56908#issuecomment-2421969826

Reply:
Thanks!